src/gpl/src/graphics.cpp

```diff
@@ -454,7 +454,8 @@ bool Graphics::populateMap()
           0.0f,
           static_cast<float>(bin.instPlacedAreaUnscaled())
               + static_cast<float>(bin.nonPlaceAreaUnscaled()) - scaledBinArea);
-      addToMap(box, value);
+      odb::dbBlock* block = pbc_->db()->getChip()->getBlock();
+      addToMap(box, block->dbuAreaToMicrons(value));
```

Comment:
@gadfort is this right or should use_dbu_ from the base HeatMapDataSource come into play?

Reply:
I think that is correct. It's just changing that information is stored in the heatmap. Which should not depend on use_dbu, however, the display and scaling code could be modified to honor that, in which case it would be easier to store the unscaled data and scale it at the end.